pyanaconda/modules/payloads/payload/rpm_ostree/flatpak_installation.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2021  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from pyanaconda.core.i18n import _
+from pyanaconda.modules.common.task import Task
+from pyanaconda.payload.errors import PayloadInstallError, FlatpakInstallError
+from pyanaconda.payload.flatpak import FlatpakPayload
+
+
+class InstallFlatpaksTask(Task):
+    """Task to install flatpaks from the local source on SilverBlue."""
+
+    def __init__(self, sysroot):
+        """Create a new task.
+
+        :param str sysroot: path to the system root
+        """
+        super().__init__()
+        self._sysroot = sysroot
+
+    @property
+    def name(self):
+        return "Install Flatpaks"
+
+    def run(self):
+        self.report_progress(_("Starting Flatpak installation"))
+
+        flatpak_payload = FlatpakPayload(self._sysroot)
```

Comment:
Please, move the `FlatpakPayload` class to a new file in `pyanaconda.modules.payload.rpm_ostree` and rename it to something like `FlatpackManager`. I was very confused by this solution until I realized that this is not the typical payload class.

Reply:
Good idea.

Reply:
And it is done.